posts/lte-update.md

```diff
@@ -0,0 +1,131 @@
+But the `Ext` in this statement is really about the Ext-groups of condensed abelian groups,
+so both `ℳ_{p'} S` and `V` are viewed as condensed abelian groups in this theorem.
+
+## Homological algebra
+
+To be able to state this theorem (e.g., to make sense of `Ext`)
+we need to develop the theory of derived functors
+for arbitrary abelian categories with enough injectives/projectives.
+This was done by Scott Morrison, who contributed definitions of `Ext` and `Tor` to mathlib about half a year ago.
+We also need to show that the category of condensed abelian groups satisfies these conditions.
+Recently, Adam Topaz finished a formalization of the fact that on an arbitrary site (= category + Grothendieck topology)
+the category of sheaves with values in a suitable abelian category is itself an abelian category.
+In particular the category of abelian sheaves, and hence condensed abelian groups, forms an abelian category.
+The fact that condensed abelian groups have enough projectives is almost done,
+so that finally the `Ext` in the statement above does no longer depend on unformalized assumptions.
+
+To be able to compute the `Ext`-groups,
+we need a library of results about homological complexes, long exact sequences, and such.
+A fundamental input is of course the snake lemma,
+and this was formalized by Riccardo Brasca and Adam Topaz in the fall.
+Andrew Yang is currently working towards a formal proof
+that the homotopy category of complexes is a pretriangulated category.
+This will give us important tools to manipulate the `Ext`-groups
+into a shape that reduces to `first_target`.
+
+## Spaces of measures
+
+As mentioned above, the space `ℳ_{p'} S` is a real condensed vector space
+that is morally a space of measures.
+One of the crucial insights in the proof is that it is a quotient of an arithmetic analogue
+that we've been calling the space of *Laurent measures*: `ℳ(S, ℤ((T))_{r'})`,
+with `(1/2)^{p'} = r'`.
+This quotient map turns out to be the cokernel of an injective endomorphism of the space of Laurent measures.
+Together, these two maps form a short exact sequence,
+which leads to a long exact sequence of `Ext`-groups.
+This quotient map sits in a short exact sequence, together with its kernel.
+Hence the vanishing of the `Ext`-groups above can be reduced to a question about similar `Ext`-groups
+but this time involving these spaces of Laurent measures.
+
+In turn, these spaces of Laurent measures naturally admit a free module as submodule,
+and the quotient is the mysterious space `Mbar r' S` that appears in `first_target`.
+Together, these two short exact sequences are crucial inputs
+that will be combined with the homological tools mentioned above
+to reduce the main theorem to `first_target`.
+
+Filippo A.E. Nuccio has been working arduously on the formalization of these results,
+which amounts to Theorem 6.9 of [Analytic.pdf](https://www.math.uni-bonn.de/people/scholze/Analytic.pdf).
+
```

Comment:
We did a lot of work in the Fall around the various categories of comphausly/profinitely filtered gadgets, which are related to these constructions. Do you want to mention these as well? I'm not so sure, since these are objects that generic mathematicians might not really care much about.

Reply:
Yeah, I agree. They might turn out to be notions that people will want to use in the future. But I didn't see how to mention them in this blogpost in a clear and concise way.